src/madhuraMJun20/scannerClass/SwapTwoNumbers.java

```diff
@@ -0,0 +1,19 @@
+package madhuraMJun20.scannerClass;
+
+/*Program 1: Swap two numbers without using third variable.*/
+public class SwapTwoNumbers {
+	void swapNumbers(int num1, int num2) {
+		System.out.println("Input -> Number 1 = " + num1 + ", Number 2 = " + num2);
+		num2 = num1 + num2;// 30
+		num1 = num2 - num1;// 20
+		num2 = num2 - num1;
+		System.out.println("Output -> Number 1 = " + num1 + ", Number 2 = " + num2);
+	}
+
+	public static void main(String[] args) {
+		SwapTwoNumbers swapTwoNumbers = new SwapTwoNumbers();
+		int num1 = 10;
```

Comment:
Code is perfect. Just one thing that I would suggest is you should pass the numbers directly to the method calling instead of declaring 2 variables and assigning the value.

Reply:
Yes. Updated the code. thanks